django_sourcerer/models.py

```diff
@@ -2,19 +2,26 @@
 
 from django_sourcerer.import_datas.intermediate_layer import Adapter
 
-Models = type(Adapter().parse_models_name(), (models.Model,), {
-    '__module__': __name__,
-})
 
+class AutoModels:
+    ModelsFromConfFile = type(Adapter().parse_models_name(), (models.Model,), {
+        '__module__': __name__,
+    })
 
-class HandleModels:
-    dic = Adapter().parse_models_values()
 
-    for k, v in dic.items():
+class HandleModel:
 
-        if v == 'str':
-            Models.add_to_class(k, models.CharField(max_length=255))
-        elif v == 'int':
-            Models.add_to_class(k, models.IntegerField())
-        elif v == 'text':
-            Models.add_to_class(k, models.TextField(max_length=255))
+    def parse_models(self):
+        dic = Adapter().parse_models_values()
+
+        for k, v in dic.items():
+
+            if v == 'str':
+                AutoModels.ModelsFromConfFile.add_to_class(k, models.CharField(max_length=255, null=True))
+            elif v == 'int':
+                AutoModels.ModelsFromConfFile.add_to_class(k, models.IntegerField(null=True))
+            elif v == 'text':
+                AutoModels.ModelsFromConfFile.add_to_class(k, models.TextField(max_length=255, null=True))
```

Comment:
You must create your model (ModelsFromConfigFile) as a local variable here. Otherwise you won't be able to have more than one model.